I'm sorry, but an uncaught exception occurred.

While running game code:
TypeError: 'NoneType' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\renpy-8.3.6-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "D:\renpy-8.3.6-sdk\renpy\main.py", line 690, in main
    i()
TypeError: 'NoneType' object is not callable

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.6.25022803
EcoCityMatch 0.1
Sat May  3 22:39:49 2025
